models/job.js

```diff
@@ -68,8 +71,7 @@ module.exports = {
      * @type {Joi}
      */
     update: Joi.object(mutate(MODEL, [], [
-        'state',
-        'containers'
+        'state'
```

Comment:
This is for jobs?

Should containers only ever be updated through the screwdriver.yaml?